File: testdata/eval/lua/sample.lua
local UserService = {}
UserService.__index = UserService

function UserService.new()
    local self = setmetatable({}, UserService)
    return self
end

function UserService:getUser(id)
    return "user-" .. tostring(id)
end

local function handleRequest()
    local svc = UserService.new()
    svc:getUser(1)
end

return UserService

File: testdata/eval/rust/sample.rs
use std::fmt;

struct UserService {
    id: i32,
}

impl UserService {
    fn get_user(&self) -> String {
        format!("user-{}", self.id)
    }
}

fn handle_request() {
    let svc = UserService { id: 1 };
    svc.get_user();
}
